src/functions.js

```diff
@@ -484,3 +484,18 @@ function news_ (tag, from, to, size) {
     ]
   }))
 }
+
+function latestPrice_ (slug, currency) {
+  var supportedCurrencies = ['USD', 'BTC']
+
+  if (supportedCurrencies.indexOf(currency.toUpperCase()) === -1) {
```

Comment:
Will `if supportedCurrencies.includes(currency.toUpperCase()) {` work?

Reply:
It won't. Works locally but not in the webeditor. 
It's introduced in [ES2016](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes#Specifications)